els/frontend/src/actions/auth.js

```diff
@@ -1,5 +1,7 @@
 import axios from 'axios';
 import { returnErrors } from './messages';
+import config from '../utils/const';
+// import tokenConfig from '../utils/helpers/authHelper';
```

Comment:
Use this import `tokenConfig` rather than creating another one with the same name here
```suggestion
import config from '../utils/const';
import { tokenConfig } from '../utils/helpers/authHelper';
```